src/modules/pointsVault/components/PointsVaultLogin.tsx

```diff
@@ -1,10 +1,151 @@
-import { Box } from 'blocks';
 import { FC } from 'react';
 
-export type PointsVaultLoginProps = {};
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+import styled from 'styled-components';
+import { MdError } from 'react-icons/md';
 
-const PointsVaultLogin: FC<PointsVaultLoginProps> = () => {
-  return <Box>Points vault login</Box>;
+import { usePointsVaultUserLogin } from 'queries';
+import useToast from 'hooks/useToast';
+
+import { Box, Button, TextInput, PushLogo, Text, BellSimple } from 'blocks';
+
+import { PointsVaultComponents } from '../PointsVault.types';
+
+export type PointsVaultLoginProps = {
+  handleSetActiveComponent: (component: PointsVaultComponents) => void;
+};
+
+const PointsVaultLogin: FC<PointsVaultLoginProps> = ({ handleSetActiveComponent }) => {
+  const { mutate: pointsVaultUserLogin, isPending } = usePointsVaultUserLogin({
+    username: '',
+    password: '',
+  });
+
+  const validationSchema = yup.object().shape({
+    username: yup.string().required('Username is required'),
+    password: yup.string().required('Password is required'),
+  });
+
+  const formik = useFormik({
```

Comment:
```suggestion
  const formik = useFormik<VaultLoginformValues>({
```

export type VaultLoginFormValues = { username: string, password:string }